rwgame/debugstate.cpp

```diff
@@ -129,6 +130,53 @@ DebugState::DebugState(RWGame* game, const glm::vec3& vp, const glm::quat& vd)
 	m->addEntry(Menu::lambda("Cull Here", [=] {
 		game->getRenderer()->setCullOverride(true, _debugCam);
 	}, entryHeight));
+	m->addEntry(Menu::lambda("Unsolid garage doors", [=] {
+
+		std::vector<std::string> garageDoorModels {
```

Comment:
It's only a debug feature (called only if you choose to do so) and std::array doesn't support initializing from an unknown number of elements afaik?
The compiler will most certainly optimize everything of that away anyway.